### PR TITLE
Readme and Spacing variable fix

### PR DESCRIPTION
**Pull request by monicawheeler**
# Description
The purpose of this work is to update the README. In the process of reviewing the documentation, I ran across a small issue with the box padding output.

This PR deals with several updates.
1. Updates the Pine README with the Pine graphic for light and dark mode.
2. Updates the README Roadmap chart to better reflect the state of the system.
3. Corrects an incorrect padding for "xxl" outputting as "2xl" and therefore not rendering the correct padding in the box example. 

| Before | After |
| -------- | ------- |
| Screenshot 2024-03-26 at 10 52 05 AM | Screenshot 2024-03-26 at 10 52 41 AM |

## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
1. To view the README, navigate to the file and select ["View File"](https://github.com/Kajabi/pine/blob/905aee587645887bb5b73fd1a66ca82ea4fa5003/README.md)
2. To view the fixed bug, navigate to http://localhost:6006/?path=/docs/components-layout-box--docs and view the updated padding for the **xxl** size box.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [ ] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

**Comment by ju-Skinner** [comment]
This will be a breaking change. @monicawheeler  You should change the commit [0962ccd](https://github.com/Kajabi/pine/pull/159/commits/0962ccdc16805c949d4b1dc2cf9c6ed8b7ef8f38) to read "[fix(spacing)!..." note the `!`. This will cause the version to increment the major.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
-
-Pine is Kajabi's UI toolkit for building beautiful, accessible web applications.
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/assets/pine-logo-dark.png">
+    <source media="(prefers-color-scheme: light)" srcset="./docs/assets/pine-logo.png">
+    <img alt="Pine: Web Component Design System" src="./docs/assets/pine-logo.png">
+  </picture>
+  <br>
+  <a href="https://main--pine-design-system.netlify.app/?path=/docs/welcome--docs">Documentation Site</a>
```
Comment (pixelflips) [inline README.md]: I may be out of date, but I have https://pine-design-system.netlify.app/ as the URL?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
-# Pine
-
-Pine is Kajabi's UI toolkit for building beautiful, accessible web applications.
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/assets/pine-logo-dark.png">
+    <source media="(prefers-color-scheme: light)" srcset="./docs/assets/pine-logo.png">
+    <img alt="Pine: Web Component Design System" src="./docs/assets/pine-logo.png">
+  </picture>
+  <br>
```
Comment (pixelflips) [inline README.md]: Could this section be refactored to two paragraph elements so the `<br> `could be dropped?

File: libs/core/src/global/styles/tokens/_line-height.scss [hunk 3]
```diff
@@ -4,7 +4,7 @@
   --pine-line-height-md: 24px;
   --pine-line-height-lg: 28px;
   --pine-line-height-xl: 32px;
-  --pine-line-height-2xl: 40px;
+  --pine-line-height-xxl: 40px;
   --pine-line-height-3xl: 48px;
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/_line-height.scss:8]: Not related to these updates, but we should decide if we want to use numbers or letters for these variations. 3xl and 4xl as examples.

**Review by pixelflips — APPROVED** [review]
Nothing blocking. A few suggestions and notes. LGTM! 👍🏼

File: libs/core/src/global/styles/tokens/_line-height.scss (hunk 3 above)
Comment (monicawheeler) [reply libs/core/src/global/styles/tokens/_line-height.scss:8]: I wonder if 3xl and 4xl is an exception and makes it more legible? Thoughts @Kajabi/dss-devs?

## With numbers
```
:root {
  --pine-line-height-xxl: 40px;
  --pine-line-height-3xl: 48px;
  --pine-line-height-4xl: 60px;
}
```

## With letters
```
:root {
  --pine-line-height-xxl: 40px;
  --pine-line-height-xxxl: 48px;
  --pine-line-height-xxxxl: 60px;
}
```

Comment (ju-Skinner) [reply libs/core/src/global/styles/tokens/_line-height.scss:8]: I like the number variation. I don't have time to count the number of `x's` LOL.

File: .tool-versions
Comment (ju-Skinner) [inline .tool-versions:1]: Did you mean to commit this? We are only on Node v20 for now.

File: libs/core/src/global/styles/tokens/_line-height.scss (hunk 3 above)
Comment (pixelflips) [reply libs/core/src/global/styles/tokens/_line-height.scss:8]: Fair enough and once we get into more semantic tokens these will change anyway.

**Review by ju-Skinner — APPROVED** [review]

**Pull request merged**